app/lib/methods/getCustomEmojis.js

```diff
@@ -84,14 +84,18 @@ export async function setCustomEmojis() {
 export function getCustomEmojis() {
 	return new Promise(async(resolve) => {
 		try {
-			const serverVersion = reduxStore.getState().server.version;
+			const { server } = reduxStore.getState().server;
+			const serversDB = database.servers;
+			const serversCollection = serversDB.collections.get('servers');
+			const serverInfo = await serversCollection.find(server);
+
```

Comment:
Is it possible to create a helper to prevent duplicate this code everywhere we need to check the version?

Reply:
Yeah! 👍

Reply:
@rodrigok what you think about it?
> Other possible approach is create a action setVersion and call it when getApiInfo 🤔 ...

Reply:
@djorkaeffalexandre I'm not sure how those actions work so can't opine on that matter :(

Reply:
Ok! Thx :)